### PR TITLE
RIP @shiftkey's inbox for the next 48 hours

### PR DESCRIPTION
**Pull request by ghuntley**

### CODE REVIEW

**Comment by khellang** [comment]
Uh, the label is still called `up-for-grabs`? https://github.com/reactiveui/website/labels/up-for-grabs

**Comment by ghuntley** [comment]
Ta @khellang. Fixed! 🚢

File: _data/projects/reactiveui-website.yml [hunk 1]
```diff
@@ -10,5 +10,5 @@ tags:
 - "Cake"
 - "Wyam"
 upforgrabs:
-  name: up-for-grabs
-  link: https://github.com/reactiveui/website/labels/up-for-grabs
+  name: help wanted
+  link: https://github.com/reactiveui/reactiveui/labels/help%20wanted
```
Comment (shiftkey) [inline _data/projects/reactiveui-website.yml]: `reactiveui/website`

**Review by ghuntley — COMMENTED** [review]
tough crowd tonight 👍 🍺

**Comment by ghuntley** [comment]
😍

**Pull request merged**